VehicleCode/Input/AnalogSensor.cpp

```diff
@@ -48,13 +46,21 @@ int AnalogSensor::readInputs() {
 /**
  * @brief Builds the data array for the CAN message.
  * 
```

Comment:
Add description of expected input and output format so that we can refer here to know how to reconstruct the returned object